assigment1.js

```diff
@@ -0,0 +1,25 @@
+//1a
+function tvNames(array){
+    return array.map((tv) => tv.name);
+}
+console.log(tvNames(inventory));
+//1b
+function outOfStockTVs(array){
+   return array.filter((tv) => tv.originalStock === tv.sold);
+}
+console.log(outOfStockTVs(inventory));
+//1c
+function ambiLightTVs (array) {
+    return array.filter((tv) => tv.options.ambiLight === true);
```

Comment:
Deze zou nog iets korter en simpeler kunnen
```suggestion
    return array.filter(tv => tv.options.ambiLight);
```